avro-converter/src/test/java/io/confluent/connect/avro/AdditionalAvroDataTest.java

```diff
@@ -0,0 +1,145 @@
+package io.confluent.connect.avro;
+
+import java.io.File;
+import java.io.IOException;
+
+import io.test.avro.core.AvroMessage;
+import io.test.avro.doc.DocTestRecord;
+import org.apache.avro.Schema;
+import org.apache.avro.Schema.Parser;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecordBuilder;
+import org.apache.avro.reflect.ReflectData;
+import org.apache.avro.reflect.Union;
+import org.apache.avro.specific.SpecificData;
+import org.apache.kafka.connect.data.SchemaAndValue;
+import org.junit.Assert;
+import org.junit.Test;
+import uk.co.jemos.podam.api.PodamFactory;
+import uk.co.jemos.podam.api.PodamFactoryImpl;
+
+public class AdditionalAvroDataTest
+{
+
+
+
+    @Test
+    public void testDocumentationPreservedSchema() throws IOException
+    {
+        AvroData avroData = new AvroData(1, false);
```

Comment:
Makes sense, will pick up when we commit in solution for breaking name (name to full name)